### DISCUSSION TITLE
Does BEM extend to ids?

### DISCUSSION THREAD

**Discussion by glennsimon** [getbem/getbem.github.io#277] [Q&A]
What is the recommendation for id naming?  I would assume that an id (#) would have the same rules as a block, but is BEM intended only for class naming and not ids?

**Comment by iamstarkov** [comment]
BEM recommends to avoid id usage as much as possible

**Comment by glennsimon** [comment]
Thank you @iamstarkov. Why does bem recommend avoiding using IDs?

**Comment by iamstarkov** [comment]
because of very high specificity
http://cssguidelin.es/#specificity
http://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/
